src/main/java/com/calc/CalculatorAllinOne/factory/ObjectFactory.java

```diff
@@ -11,6 +12,7 @@ public static Calculator getOperationObject(String choiceName) {
 		Addition addition = new Addition();
 		Substraction substraction = new Substraction();
 		Multiplication multiplication = new Multiplication();
+		Division division=new Division();
```

Comment:
@AjitDasComputerScience  Please check this once . I think you have not yet comitted Division class .